### ISSUE TITLE
Reverse image search w/ google stopped working recently

### ISSUE THREAD

**Issue by davidme80** [rohitbatra1/Safari-Reverse-Image-Search#9]
When selecting image search with google, this does not bring up results. Only the google search page appears. Tin eye and yandex work.  Safari ver: Version 16.1 (18614.2.9.1.12)

**Comment by soundsgoodtome** [comment]
Yes. Same issue here. Google has made changes that have broken the extension. Interestingly, I have a similar extension for Firefox, called "Search by Image on Google," which also broke recently. That developer has already issued a fix. Developer: Please fix this Safari extension, as well.
